custom_components/meross_cloud/cover.py

```diff
@@ -128,7 +134,7 @@ def supported_features(self):
         """Flag supported features."""
         # So far, the Roller Shutter RST100 supports position, but it looks like it is fake and not reliable.
         # So we don't support that on HA neither.
-        return CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE | CoverEntityFeature.STOP
+        return CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE | CoverEntityFeature.STOP | CoverEntityFeature.SET_POSITION
```

Comment:
Before adding this flag, we should find a way to determine if the underlying Meross device really supports setting the intermediate cover position. As per my findings, HW revision 2.0 does not support such a feature. 

Why don't you use the[ MerossIot sniffer tool](https://albertogeniola.github.io/MerossIot/advanced-topics.html#sniff-device-data) to capture the info from your device? This might help us in determing how the Meross device reports this capability to the app, and therefore implement the same check in the low-level library.

Reply:
I will report it as soon as I do it. Thanks @albertogeniola

Reply:
please find data.zip attached. I replaced sensitive data like IP info, MAC, userId... 
[data.zip](https://github.com/albertogeniola/meross-homeassistant/files/15000350/data.zip)

I hope it helps

Reply:
after https://github.com/albertogeniola/meross-homeassistant/pull/496/commits/71f1dc1d9494f3f6a5d678e606b3569e34a48325 and https://github.com/albertogeniola/meross-homeassistant/pull/496/commits/8de0937c03d4c206fd50920b1d265874ae1182e9 initial position is properly retrieved.

Many thanks @albertogeniola 
I don't know whether `meross_iot` `0.4.7.0b1` is a final version, otherwise please let me know what's the one and I will adapt this PR

Reply:
Great to hear that. No, that's not the final version, but it is stable enough. Let me perform some more checks and I'll release a stable version soon.